src/components/common/GuideCard/GuideCard.jsx

```diff
@@ -1,32 +1,187 @@
+            backgroundSize: 'cover',
+            backgroundPosition: 'center',
+          }}
+        >
+          <div className="flex justify-between">
+            <div
+              className={cn(
+                'flex size-8 justify-center rounded-br-lg rounded-tl-lg bg-gray-300/60',
+                isLiked ? 'w-10' : null,
+              )}
+            >
+              <button
+                className="flex items-center fill-current p-1.5"
+                onClick={handleLikeClick}
+                aria-label={
+                  isLiked
+                    ? 'Remove like from this guide'
+                    : 'Add like to this guide'
+                }
+              >
+                <Icon
+                  name="heart"
+                  size="24px"
+                  className={cn(
+                    'fill-current stroke-none hover:text-orange-color',
+                    isLiked
+                      ? 'text-xs text-orange-color'
+                      : 'text-dark-color/80',
+                  )}
+                />
+                {likesAmount > 0 && (
+                  <span className="mx-1 pt-1 text-lg font-semibold text-orange-color">
+                    {likesAmount}
+                  </span>
+                )}
+              </button>
+            </div>
+            {isHovered ? (
+              <div className="my-auto ml-2 w-9/12 text-left font-fourth text-lg font-light italic">
+                <p>Voyager</p>
+              </div>
+            ) : null}
+            <div className="flex size-8 justify-center rounded-bl-lg rounded-tr-lg bg-gray-300/60">
+              <button
+                className="fill-current"
+                onClick={handleShareClick}
+                aria-label="Share this guide"
+              >
+                <Icon
+                  name="link"
+                  size="21px"
+                  strokeWidth={3}
+                  className="text-dark-color/80 hover:text-orange-color"
+                />
+              </button>
+            </div>
+          </div>
+          {isHovered ? (
+            <div className="flex h-full w-full flex-col justify-between rounded-b-lg bg-light-color/90 p-2.5">
+              <div className="mx-3.5 mt-2.5">
+                <p
+                  title={guideTitle}
+                  className="line-clamp-3 break-words font-fourth text-lg font-medium"
+                >
+                  {guideTitle}
+                </p>
+                <p title={guideText} className="line-clamp-6 font-light">
+                  {guideText}
+                </p>
+              </div>
+              <Link
+                to={`/cities/guides/${guideId}`}
+                className="mx-3.5 underline hover:text-orange-color"
+              >
+                <p className="inline-block font-fourth text-lg font-medium">
+                  Read More
+                </p>
+                <ErrorBoundary FallbackComponent={FallbackIcon}>
```

Comment:
No need wrap the component to ErrorBoundary if we know that it won't throw an error